libs/utils/utils.js

```diff
@@ -251,6 +251,22 @@ export const [setConfig, updateConfig, getConfig] = (() => {
   ];
 })();
 
+export function getSusiOptions() {
+  const { susiOptions, env: { name: envName } } = getConfig();
```

Comment:
I'd define susiOptions on the consuming projects with the appropiate stage or prod config, then you don't need to check it in this function. And you can create a default for it to be an empty object if undefined is not wished.

Then the `getSusiOptions` get be simplified to `getConfig().susiOptions` and you save the 15 lines of helper function

Reply:
Sounds good!